novuhq/website
# feat: use-case pages polishing

This pull request brings some refactoring for usecases pages.

Steps to test:

- open preview
- check that everything works as expected
